pom.xml

```diff
@@ -176,6 +176,12 @@ THE SOFTWARE.
         <artifactId>json</artifactId>
         <version>20190722</version>
       </dependency>
+      <dependency>
+        <groupId>commons-beanutils</groupId>
+        <artifactId>commons-beanutils</artifactId>
+        <!-- 1.9.4 when this merged https://github.com/stapler/stapler/pull/211 -->
```

Comment:
Why wait?

Reply:
I also don't understand this. Is this needed by common-digister3?

Reply:
nope can be removed. commons-beanutils 1.9.4 is now supported by core